README.md

```diff
@@ -1,3 +1,5 @@
-# ndt7-client-golang
+[![GoDoc](https://godoc.org/github.com/m-lab/ndt7-client-go?status.svg)](https://godoc.org/github.com/m-lab/ndt7-client-go) [![Build Status](https://travis-ci.org/m-lab/ndt7-client-go.svg?branch=master)](https://travis-ci.org/m-lab/ndt7-client-go) [![Coverage Status](https://coveralls.io/repos/github/m-lab/ndt7-client-go/badge.svg?branch=master)](https://coveralls.io/github/m-lab/ndt7-client-go?branch=master) [![Go Report Card](https://goreportcard.com/badge/github.com/m-lab/ndt7-client-go)](https://goreportcard.com/report/github.com/m-lab/ndt7-client-go)
 
-Reference ndt7 golang client implementation.
+# ndt7 Go client
+
+Reference ndt7 Go client implementation.
```

Comment:
After this PR is merged, I'll write a `main()` with specific pointers to godoc.org.

Reply:
https://github.com/m-lab/ndt7-client-go/issues/6